src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -118,7 +121,10 @@
           AfterTargets="CheckTestCategories"
           Inputs="@(RunTestsForProjectInputs)"
           Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestTargetFramework.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
-          Condition="'$(TestDisabled)'!='true'">
+          >
+    <GetTargetMachineInfo>
+      <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
```

Comment:
If I have a test that is not platform specific (e.g it would go into the AnyOS folder) what actually happens here?  Do we end up generating a runner script with xunit arguments to filter out tests based on the current system?

BTW, this task has a very confusing name, since it actually get's information about the host doing the build (but I know you are just reusing an existing task)